docker/images/alpine/Dockerfile

```diff
@@ -0,0 +1,103 @@
+# VideoFront, a FUN LTI video provider
+#
+# Nota bene:
+#
+# this container expects two volumes for statics and media files (that will be
+# served by nginx):
+#
+# * /data/media
+# * /data/static
+#
+# Once mounted, you will need to collect static files via the eponym django
+# admin command:
+#
+#     python ./manage.py collectstatic
+#
+
+# ---- base image to inherit from ----
+FROM python:3.6-alpine as base
+
+# ---- builder image ----
+FROM base as back-builder
+
+# Install development libraries required for compiled python dependencies (lxml,
+# Pillow and psycopg2)
+RUN apk --no-cache add --update \
+        # lxml dependencies
+        libxml2-dev \
+        libxslt-dev \
```

Comment:
Can you remind me which dependency requires lxml?

Reply:
pycaption==1.0.0
  - beautifulsoup4 [required: >=4.2.1,<4.5.0, installed: 4.4.1]
  - cssutils [required: >=0.9.10, installed: 1.0.2]
  - enum34 [required: Any, installed: 1.1.6]
  - future [required: Any, installed: 0.16.0]
  - lxml [required: >=3.2.3, installed: 4.2.2]
  - six [required: >=1.9.0, installed: 1.11.0]